pom.xml

```diff
@@ -0,0 +1,66 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.simple</groupId>
+  <artifactId>kafka-dropwizard-reporter</artifactId>
+  <version>1.0-SNAPSHOT</version>
+  <packaging>jar</packaging>
+
+  <name>kafka-dropwizard-reporter</name>
+  <url>https://github.com/SimpleFinance/kafka-dropwizard-reporter</url>
+  <description>Register built-in Kafka client metrics to Dropwizard Metrics</description>
+
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <maven.compiler.source>1.7</maven.compiler.source>
+    <maven.compiler.target>1.7</maven.compiler.target>
+  </properties>
+
+  <developers>
+      <developer>
+          <name>Jeff Klukas</name>
+          <email>jeff@klukas.net</email>
```

Comment:
It's really your personal preference regarding how comfortable you are with your email being out there for support emails.